### DISCUSSION TITLE
Octoprint backup file location on android?

### DISCUSSION THREAD

**Discussion by Madcamper** [feelfreelinux/octo4a#434] [Q&A]
I am trying to have an automated process of taking backups of my octoprint config and then uploading to my NAS. I've currently got the "Backup Scheduler" plugin installed which routinely creates the backups. The only other plugins that are available are a Webdav plugin(which is what I really need), a one drive backup and a google drive backup. None of these plugins will successfully install on octo4a due to their dependencies on specific packages that can't be(to my knowledge) installed on android.
My next route was to find the source of the backup file in the android file explorer and using my file sync android app of choice(synology) and sync it to my NAS that way. I've browsed through the octo4a app folder via the file explorer in android and through SSH via octoprint with no success.
Does anyone know where the backup files that are created from the backup method in octoprint, is stored on android?

**Comment by joaomnuno** [comment]
The backups are created under the directory: ~/.octoprint/data/backup
I see you are having issues with the plugins. It's true that some plugins don't fully work on android, however make sure you have the plugins extra installed on the octo4a app
![image](https://github.com/feelfreelinux/octo4a/assets/49878220/00cf6e60-a434-4b90-a453-8ba6fc43dd81)

**Comment by Madcamper** [comment]
Thanks for commenting! I checked the ~/.octoprint/data/backup location before and nothing was there. Now when I checked, I see the backup file there and was able to setup a backup to my synology NAS using FolderSync android app and the webdav protocol. I do also have the Extra plugins installed.

**Comment by joaomnuno** [comment]
You're very welcome! I'm so glad I could help! Communities like these are all about helping each other out. If you have any more questions or need further assistance in the future, don't hesitate to ask.

Happy printing!
